README.md

```diff
@@ -204,3 +204,59 @@ The following parameters refer to components that are not internal to Codacy, bu
 You can also configure values for the PostgreSQL database via the Postgresql [README.md](https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
 
 For overriding variables see: [Customizing the chart](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing)
+
+## Development
+
+### Set up your environment for DigitalOcean clusters
+
+### 1. Requirements
+
+Using a DOKS cluster has the following requirements:
+
+-   [doctl](https://github.com/digitalocean/doctl)
+-   [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+On macOS you can install all these tools with brew:
+
+```bash
+$ brew install doctl kubectl
+```
+
+### 2. Configuring the DigitalOcean cli
+
+To configure the cli create a [personal access token](https://www.digitalocean.com/docs/api/create-personal-access-token/), then run:
+
+```bash
+$ doctl auth init
+```
+
+### 3. Setup your kubecontext for the target
+
+Replace the `<codacy-doks-cluster-hourly>` with the name of your target cluster.
+
+```bash
+$ doctl kubernetes cluster kubeconfig save <codacy-doks-cluster-hourly> --set-current-context
+```
+
+## Deploy your version of a component
+
+You can deploy your version of a component using any of the `.circleci`
+pipelines we have set up in this project. Currently, we use the `hourly_pipeline`
+for this purpose.
+
+1.  Checkout this repository
+2.  Create a new branch
+3.  Change the `requirements.yaml` to use your version
+4.  `helm dep up`
+5.  Change the `.circleci`  to deploy your branch on the workflow called `hourly_pipeline` (we need to rename this)
+6.  `git push`
+7.  Follow the circleci pipeline and use kubectl to see if your installation was successful
+8.  Go to <http://k8s.hourly.dev.codacy.org/> and test it out
```

Comment:
Isn't it necessary to merge the new branch back to master at the end?

Reply:
No, the if you change `[here]( https://github.com/codacy/chart/blob/master/.circleci/config.yml#L524)` and add your branch, the circleci will run for your branch. You can just mess around in your branch.

Your installation will only be overwritten when someone decides to do the same, only one person at a time can be doing this.